src/backend/distributed/Makefile

```diff
@@ -5,7 +5,7 @@ citus_top_builddir = ../../..
 
 MODULE_big = citus
 EXTENSION = citus
-EXTVERSIONS = 5.0 5.0-1 5.0-2
+EXTVERSIONS = 5.0 5.0-1 5.0-2  5.1-1
```

Comment:
Oh, and I assume `EXTVERSIONS` has no special meaning? Couldn't find anything with a quick Google.